packages/opentelemetry-test-utils/src/test-fixtures.ts

```diff
@@ -42,7 +42,7 @@ import type { InstrumentationOption } from '@opentelemetry/instrumentation';
  */
 export function createTestNodeSdk(opts: {
   serviceName?: string;
-  instrumentations: InstrumentationOption[];
+  instrumentations: (Instrumentation | Instrumentation[])[]
```

Comment:
Reviewer note: this was https://github.com/open-telemetry/opentelemetry-js/pull/4675